### PR TITLE
PIM-5991: Add the focus into wysiwig field

### PR DESCRIPTION
**Pull request by anaelChardan**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md) 

**Description (for Contributor and Core Developer)**

[//]: <> (What does this Pull Request do? reference the related issue?)

Create a textarea attribute and enable wysiwyg mode. 
Add the attribute to a family and mark it as required for completeness. 
Edit a product of the family. 
Open the completeness tab. 
Click on the link corresponding to the empty wysiwyg field. 
Add the focus into the concerned field

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       |:negative_squared_cross_mark:
| Added Behats                      | :negative_squared_cross_mark:
| Changelog updated                 | :white_check_mark:
| Review and 2 GTM                  | :white_check_mark:
| Micro Demo to the PO (Story only) | :negative_squared_cross_mark:
| Migration script                  |  :negative_squared_cross_mark:
| Tech Doc                          | :negative_squared_cross_mark:

:white_check_mark: Done and pass

:red_circle: Done but fail

:clock1: Done but pending

:negative_squared_cross_mark: Not needed

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/wysiwyg-field.js [hunk 1]
```diff
@@ -45,6 +45,9 @@ define(
                 data = '' === data ? this.attribute.empty_value : data;
 
                 this.setCurrentValue(data);
+            },
+            setFocus: function () {
```
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/wysiwyg-field.js]: please add blank line before and jsdoc

Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/wysiwyg-field.js]: You can add jsdoc to this file too :)

File: CHANGELOG-1.5.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ## Bug fixes
 
 - PIM-6026: Fix an error on family mass edit that occurs when working on more families than the mass edit batch size
+- PIM-5991: Add the focus into wysiwig field
```
Comment (phaseinducer) [inline CHANGELOG-1.5.md]: Not very clear. Maybe "Focus on wysiwig field when clicking on the product edit form completeness link" ?

Comment (anaelChardan) [inline CHANGELOG-1.5.md]: Agree :)

**Pull request merged**